source/nodejs/adaptivecards-designer-app/src/app.ts

```diff
@@ -17,6 +17,7 @@ window.onload = function () {
     ACDesigner.GlobalSettings.enableDataBindingSupport = true;
     ACDesigner.GlobalSettings.showDataStructureToolbox = false;
     ACDesigner.GlobalSettings.showSampleDataEditorToolbox = true;
+    ACDesigner.GlobalSettings.showSampleHostDataEditorToolbox = true;
 
     // Uncomment to configure default toolbox titles
```

Comment:
When new card, the editor at least should have `{}` by default instead of empty.
![image](https://user-images.githubusercontent.com/6290692/180305349-729b11a7-d0f0-4bc5-8772-8f1a67e76044.png)